Makefile

```diff
@@ -16,15 +16,32 @@
 # LDFLAGS       linker flags for linking all binaries
 #
 
-SRC = $(wildcard c_src/*.c)
-HEADERS = $(wildcard c_src/*.h)
+SRC = c_src/sqlite3_nif.c
+HEADERS = c_src/utf8.h
+
+ifeq ($(ESQLITE_USE_SYSTEM),)
```

Comment:
We'll want to change this to `EXQLITE_USE_SYSTEM`

Reply:
oops, muscle memory kicked in. Updating..